### ISSUE TITLE
"maxdisp" in initialization_loss.py

### ISSUE THREAD

**Issue by wbensvage** [MJITG/PyTorch-HITNet-Hierarchical-Iterative-Tile-Refinement-Network-for-Real-time-Stereo-Matching#3]
there's a "maxdisp" in "subpix_cost" function in "loss/initialization_loss.py",  line 28 to be accurate. But it seems to be a bug? It hasn't been used as a hyperparameter, nor a passed temporal parameter.

**Comment by MJITG** [comment]
Thanks for reporting this bug. It's fixed now.
